### PR TITLE
EIC console - add reviewer rating

### PR DESCRIPTION
**Pull request by xkopenreview**
add reviewer rating in review progress column under reviewer status tab
add a rating column under reviewer status tab

### CODE REVIEW

**Comment by xkopenreview** [comment]
image

File: openreview/journal/webfield/editorsInChiefWebfield.js [hunk 1]
```diff
@@ -656,7 +687,7 @@ var renderData = function(venueStatusData) {
       papersAssigned: ['reviewerProgressData.numPapers'],
       default: ['summary.name']
     },
-    extraClasses: 'console-table'
+    extraClasses: 'eic-console-table'
```
Comment (melisabok) [inline openreview/journal/webfield/editorsInChiefWebfield.js:690]: why did you have to change this class name?

Comment (xkopenreview) [reply openreview/journal/webfield/editorsInChiefWebfield.js]: so that the column width can be overwritten

Comment (xkopenreview) [reply openreview/journal/webfield/editorsInChiefWebfield.js]: moved to element
https://github.com/openreview/openreview-py/pull/1132/commits/04681f29ff7a8ae7dc93338006d4a6cb2e9d941c

Comment (melisabok) [reply openreview/journal/webfield/editorsInChiefWebfield.js]: yes, in the post render table

**Comment by melisabok** [comment]
Is there a way to show the counters in the same line always?
Screen Shot 2022-02-16 at 12 37 08 PM

**Comment by xkopenreview** [comment]
> Is there a way to show the counters in the same line always? Screen Shot 2022-02-16 at 12 37 08 PM

please try [0b80483](https://github.com/openreview/openreview-py/pull/1132/commits/0b80483d04808163fca0f0d6356ef77735518faa)

**Comment by xkopenreview** [comment]
average rating is added in [8e6eeb8](https://github.com/openreview/openreview-py/pull/1132/commits/8e6eeb8660d3e1c49a22343022c6bfd18f3b709e)

it assumes that the rating text used in reviewer rating invitations (and thus in the dropdown in note editor) are all the same and is the same as in the map defined in EIC webfield

**Review by melisabok — APPROVED** [review]

**Pull request merged**